### PR TITLE
Gamerzone Mobile and Desktop Event page

### PR DESCRIPTION
**Pull request by fitz95**
- desktop page 
- mobile page
- about page
- home page 
- A link presenting the project overview

https://www.loom.com/share/29cb6d04cf8545f6afd2f185514bcd4f

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,26 +1,203 @@
...
+            </p>
+          </div>
+          <div>
+            <h1 class="thirdheader">2022.10.14(THUR) ~ 16(FRI)</h1>
+            <p class="secondparagraph">
+              The next gamerzone event that will be held at Mainz will feature many 
+              activities at the Mainz center  as well as meeting renowned gamers live.
+            </p>
+          </div>
+        </section>
```
Comment (mwanawabangona) [inline index.html:97]: - [ ] `required`
- [ ] Amazing job so far 👏 
- [ ] however there a few things you need to adjust
- [ ] The `Fonts` and `spacing`
- [ ] please move everything more to the right, 
- [ ] Also try to make the Layout: composition and space between elements, more like the provided design
- [ ] As the requirements states `The pages should look almost identical to the original design.`
- [ ] please see screenshots below for further comparison

`your design`
image

`Behance design`
image

File: index.html [hunk 2]
```diff
@@ -1,26 +1,203 @@
...
+            </div>
+            <div class="programflex">
+              <i class="fa fa-empire fa-3x" aria-hidden="true"></i>
+              <div class="programh2"><h3>Empire</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will be much 
+            </div>
+          </div>
+          <button class="programbtn">JOIN GAMERZONE EVENT 2022 </button>
+        </section>
```
Comment (mwanawabangona) [inline index.html:146]: - [ ] `requred`
- [ ] Kindly make the border boxes like the original design
- [ ] also please change the background, to the original on so as to follow the original design.
- [ ] Please adjust the font of  `main program`
- [ ] please see the screenshots below

- [ ] your design 
image

- [ ] Behance design
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+              learn from the leading Gamers and added tips how to make money by being a gamer.
+            </p>
+          </div>
+          <div>
+            <p class="paragraph-mobile-about center">
+              Please do well to contact us using this email.<br />
+              <b>fitzgeraldnsong@gmail.com</b>
+            </p>
+          </div>
+        </section>
```
Comment (mwanawabangona) [inline about.html]: - [ ] `required`

- [ ] Please adjust the typography, Fonts and spacing according to the design
- [ ] i have highlighted things which needs to be adjusted 
- [ ] please see screenshots below for comparisons

- [ ] Your deign
image

- [ ]  Behance design
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @fitz95 ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @fitz95, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- GitHub flow ✔️
- Commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**